docs/HACKING.md

```diff
@@ -24,3 +24,11 @@ pip install --user -e .
 ```console
 python3.7 setup.py test
 ```
+# Check the Report for Unit Test Coverage
+```
+cd dffml
```

Comment:
nit: Lets leave this `cd dffml` out due to there being a `dffml` directory in the top level `dffml` directory, and this may confuse users as most command line examples are assumed to be done from that top level `dffml` directory